code/game/objects/effects/landmarks/landmarks.dm

```diff
@@ -56,10 +56,8 @@
 		qdel(src)
 
 
-/obj/effect/landmark/newplayer_start/Initialize()
-	. = ..()
+/obj/effect/landmark/newplayer_start/New() //This can't be Initialize() or players will start in a wrong loc at roundstart.
```

Comment:
change it to do `GLOB.newplayer_start += src` then the newplayer code to do `get_turf(pick(GLOB.newplayer_start))`

Reply:
Done.